scene/gui/control.cpp

```diff
@@ -475,7 +479,9 @@ void Control::_notification(int p_notification) {
 		} break;
 		case NOTIFICATION_EXIT_TREE: {
 			get_viewport()->_gui_remove_control(this);
-
+		} break;
+		case NOTIFICATION_READY: {
+			connect("ready", callable_mp(this, &Control::_clear_size_warning), varray(), CONNECT_DEFERRED | CONNECT_ONESHOT);
```

Comment:
How can this be ported to 3.2? The old version of signals prints errors.
`connect("ready", this, "_clear_size_warning", varray(), CONNECT_DEFERRED | CONNECT_ONESHOT);`

Reply:
`_clear_size_warning` needs to be bound to work in 3.2. It's one additional line in `bind_methods`.